### DISCUSSION TITLE
Same 2TB M2 for temp1 & temp2

### DISCUSSION THREAD

**Discussion by nonogrub** [madMAx43v3r/chia-plotter#142] [Q&A] [answered]
First of all thank you very much for your great Job!. Appreciate so much.
I have only one M2 of 2TB what im using for temp1 & temp2. Using this config, do you think is a good choice compare to use the official chia plotter. Any benefit?. I suppose I will have the same problem with the degradation of the M2?

Again, great JOB!!! And thanks in advance for your answer.

**Comment by casa10** [comment, marked as answer]
Degradation of the M2 in your case (tmp1 + tmp2 on the SSD) : same as using chia gui, swar or plotman.
What i understood, if you have 128GiB of DDR RAM you will use less your M2 because one part of work in RAM only.
Time to product plot using your configuration : if nvme you will divide by 2 or 3. I tested, before i need 7hr for one plot with current system on my computer. Now with this one only 2h30 using only M2 for tmp1&tmp2. (8 threads).
